src/site/_includes/components/button/button.html

```diff
@@ -1,4 +1,4 @@
-{% macro button(item) -%}
+{% macro component(item) -%}
```

Comment:
Why is there a dash before the `%`?

Reply:
No clue was there before, didn't even notice it when changing the name